JSDemos/Demos/ColorBox/Overview/description.md

```diff
@@ -1 +1,31 @@
-The ColorBox is&nbsp;a&nbsp;UI component that allows an&nbsp;end user to&nbsp;enter a&nbsp;color or&nbsp;pick it&nbsp;out from the drop-down editor.
\ No newline at end of file
+The ColorBox allows users to enter a color or pick it out from the drop-down editor. When you create a ColorBox, specify its [value]() property to set an initial color. Users can change the value instantly after they picked a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 
```

Comment:
```suggestion
The ColorBox allows users to enter a color or select it in the drop-down editor. When you create a ColorBox, specify its [value]() property to set the initial color. Users can change the value after they pick a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 
```

What do you mean when you say **control**  the behavior?

Reply:
There are two modes: one called "instantly", when users pick a shade and its code automatically appears in the input field; the other one called "useButtons", when users pick a shade and then they need to click the OK button for code to appear in the input field. When i say "control this behavior", I mean that you can pick one of the two modes and set the applyValueMode property to "instantly" or "useButtons".

Reply:
Perhaps:

To specify the mode, use the [applyValueMode]() property.